### PR TITLE
Remove redux-actions.

### LINKED ISSUE #25: Remove redux-actions dependency
### Description
The only method being used in `redux-actions` is `handleActions` which converts the reducer object into a function that receives `state, action`. Additionally, `redux-actions#handleActions` runs at `O(n)`[1]. 

We can remake it with at `O(1)` by using hash lookup. We should replicate this internally to remove the dependency and improve performance.

[1] https://github.com/acdlite/redux-actions/blob/master/src/handleActions.js#L7
  `reduce-reducers` iterates through all the reducer functions and executes them
[2] https://github.com/acdlite/redux-actions/blob/master/src/handleAction.js#L8
  `handleAction` returns state if the type doesn't match

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**
Closes #25

### CODE REVIEW

**Comment by wtgtybhertgeghgtwtg** [comment]
Since `createReducer` now returns a function that starts with a default argument (`state = Map()`), it's `length` is 0 instead of 2.  So, while it does take 2 arguments, the test will not reflect that.

File: src/createModule/formatConstants.js [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-  (modulePrefix, transform) => {
-    return {
-      ... transform,
-      formattedConstant: _formatConstant(modulePrefix, transform.action),
-    };
-  }
+const appendFormattedConstant = curry(
+  (modulePrefix, transform) => ({
+    ...transform,
+    formattedConstant: `${modulePrefix}/${transform.action}`,
```
Comment (mboperator) [inline src/createModule/formatConstants.js]: This is much cleaner

**Comment by mboperator** [comment]
+1 after the failing spec is removed. I don't think it's necessary since we actually test reducer functionality in the following lines

**Comment by wtgtybhertgeghgtwtg** [comment]
Failing spec removed.

**Comment by mboperator** [comment]
+1

**Pull request merged**